src/pages/spawn/utils.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * AZToRegion takes an availability zone and returns its region name.
+ * An AWS region is just the availability zone minus the final letter.
+ * This rule is defined in evergreen:
+ * https://github.com/evergreen-ci/evergreen/blob/299bf20c8da9e353b1097183672cd5169ff3447d/cloud/ec2_util.go#L76-L80
```

Comment:
Based on the references in Evergreen, I get the impression that this function is more of an internal function to create spawn hosts or volumes rather than something to surface to the users. 🤔 I could be wrong though!!!

Reply:
It is internal, but the purpose is to convert the volume's existing availability zone to a region that is assigned to a host. Interestingly we never expose a host's region in evergreen, only its availability zone (which is not a valid param for configuring a host). Selecting a region is pretty similar to the spawn host modal though, which is really what the migrate volume modal is (it's really not a volume configuration modal since the volume already exists). Let me know if you still feel like details are a little murky!!